src/Utils/Helpers.spec.ts

```diff
@@ -1,5 +1,8 @@
 import { Helpers } from './Helpers';
 
+// for mocking private static function Helpers.win() later
+const actualHelpersModule = jest.requireActual('./Helpers');
```

Comment:
There's no need to mock private method here. Also, we don't usually mock private members as we try to stick to black box testing